config/seed/fixtures/posts.yml

```diff
@@ -0,0 +1,180 @@
+---
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: texto-principal
+  title: Texto principal
+  markdown: << La innovación es lo que distingue a los líderes de los \nseguidores >>.- Steve Jobs
+  html: <p>&lt;&lt; La innovación es lo que distingue a los líderes de los <br />seguidores >>.- Steve Jobs</p>
+
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  title: "Dirección:"
+  slug: contactos-direccion
+  markdown: San Cayatano alto, calle París\ninstalaciones UTPL. Loja_Ecuador
+  html: <p>San Cayatano alto, calle París<br />instalaciones UTPL. Loja_Ecuador</p>
+
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  title: "Fono:"
+  featured: false
+  language: en_US
+  status: published
+  slug: contactos-fono
+  markdown: "(593 7)2585700\n(593 7)2585706\nEmail: info@fedes.ec"
+  html: "<p>(593 7)2585700</p><p>(593 7)2585706</p><p>Email: info@fedes.ec</p>"
+
+# static pages
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: true
```

Comment:
gracias por probar!

> In sqlite3, boolean value is not supported. So rails use char 't' and 'f' to represent boolean value true and false. while in other database like mysql, postgresql, the real boolean value true and false is used.

http://stackoverflow.com/questions/23223821/sqlite-boolean-t-and-f-with-rails-active-record

voy a arreglar esto y mergemos 👍